Makefile

```diff
@@ -34,10 +35,14 @@ build_cli:
 	@echo "+ $@"
 	${GO} build -o ${CLI} ${GOCMD}/cli
 
+build_contracts:
+	@echo "+ $@"
+	cd ./blockchain && npm install && ${TRUFFLE} compile
```

Comment:
yeap, its aimed to blank installation, in other case - npm return true after check